composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "more-cores/restcord",
```

Comment:
Yes it is... this repo isn't the app, it's a small composer package that bootstraps the DiscordClient in a Laravel-friendly way.